yarn-project/simulator/src/avm/avm_simulator.test.ts

```diff
@@ -389,16 +387,6 @@ describe('AVM simulator: transpiled Noir contracts', () => {
         resolveAvmTestContractAssertionMessage('u128_addition_overflow', results.revertReason!, results.output),
       ).toMatch('attempt to add with overflow');
     });
-
-    it('Expect failure on U128::from_integer() overflow', async () => {
```

Comment:
Removed this test because `from_integer` function does not exist on the native u128 type and when casting from Field to u128 there is no overflow check.